go.mod

```diff
@@ -308,9 +308,9 @@ require (
 	lukechampine.com/blake3 v1.3.0 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101603.3-0.20251024190759-96738d21616c
+// replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101603.3-0.20251024190759-96738d21616c
 
-// replace github.com/ethereum/go-ethereum => ../op-geth
+replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101603.3-rc.1.0.20251027182515-f17f8620e9c2
```

Comment:
I'll just leave a comment here to soft block merging the PR until we update this.

Reply:
Updated to the latest op-geth now.